plugins/git/plugin.zsh

```diff
@@ -150,6 +153,12 @@ geometry_prompt_git_check() {
 }
 
 geometry_prompt_git_render() {
+  # Check if we are in a bare repo
+  if [[ $(command git rev-parse --is-inside-work-tree 2>/dev/null) != "true" ]] ; then
```

Comment:
A bit of nitpicking but this `!= "true"` is kind of itching me. According to the docs, it returns `"false"` otherwise. Can we test for that?